docs/source/azure/contribute.rst

```diff
@@ -13,7 +13,7 @@ in the makefile:
 
 This creates a virtual env in your enlistment and installs all packages as editable.
 
-Instead, you can do `pip install -r tools/c7n_azure/requirements.txt` to install test dependencies.
+You can also simply do `pip install -r requirements-dev.txt` to install all dev/test dependencies.
```

Comment:
unrelated to the PR.. requirements-dev.txt doesn't install c7n, right? We can add it I think

Reply:
But C7N is in most of the plugins setup.py I think, so it ought to get installed when they are.